tests/Event/Productions/ProductionCommandHandlerTest.php

```diff
@@ -146,8 +185,24 @@ public function it_cannot_add_an_event_that_already_belongs_to_another_productio
     /**
      * @test
      */
-    public function it_can_remove_an_event_from_aproduction(): void
+    public function it_cannot_add_a_non_existing_event_to_a_production(): void
     {
+        $eventId = Uuid::uuid4()->toString();
+        $this->eventRepository->method('get')->with(...[$eventId])->willReturn(null);
```

Comment:
Is it not possible to just pass `$eventId`? No need to wrap and unwrap the array.